### PR TITLE
Remove IconName class mentions

### PR DESCRIPTION
**Pull request by marin-bratanov**

### CODE REVIEW

File: components/contextmenu/navigation.md [hunk 1]
```diff
@@ -47,26 +47,26 @@ To use the Context Menu for navigating between pages:
...
                         Url = "/profile",
-                        Icon = IconName.User
+                        Icon = "user"
                     },
                     new MenuModel()
                     {
                         Text = "Language Settings",
                         Url = "/language",
-                        Icon = IconName.Table
+                        Icon = "globe"
```
Comment (ntacheva) [inline components/contextmenu/navigation.md:69]: Table or globe?

Comment (marin-bratanov) [reply components/contextmenu/navigation.md:69]: "globe", I've messed something up, please commit the fix

Comment (svdimitr) [reply components/contextmenu/navigation.md:69]: According to me, it should be Globe, it is more suitable for the `Language setting`

Comment (svdimitr) [reply components/contextmenu/navigation.md:69]: it is globe right now, so nothing should be further committed. Not so long ago I ran this specific example

**Review by ntacheva — APPROVED** [review]

**Review by svdimitr — APPROVED** [review]

**Pull request merged**